AndyClifton/accessibility
# add minimal example to simplify debugging

### Why this new example?
They say a pull request is better than opening an issue. So here is my first pull request. The examples already provided are good but since the package is in an early stage they produce so many errors that it's hard to debug. I think I am also trying to change the scope of this package a bit. The creators until now have tried to create a package that creates tagged PDF only. However, accessibility is more than tags. We must i.e. provide a document title and a native language. Several attempts like pdfx and xmpincl have ceased to be developed so my hope is now on you, Andy :-)

### How to use the example
The idea is to run this example as the first test. If it fails the other tests will also fail. It is also intended by me that with later development of accessibility package the example can be shortened. I.e. including the `xmp` metadata for accessibility should be moved inside the package same as with `pdfdisplaydoctitle`. Also the package should run correctly with the default options which it does not as of today.

### How to verify compliance
Some tests can be automated but we should not forget the people we are doing this for so manual testing should also be done.
1. Run PAC, should not return errors
1. Run Callas PdfGoHTML, there should be the lorem ipsum text with correct spacing and no special characters
1. Enable reflow in Acrobat. Something should happen.

### Known issuses
When running the example I get two warnings:
1. `Package accessibility Warning: The chosen language (latin) is not supported by Adobe Reader 6.0. on input line 17.` I get this error for all tested languages such as english, german and latin.
1. `pdfTeX warning (dest): Page 3 has been referenced but does not exist!`

There is a clash between `xmpincl` and `hyperxmp`. Without xmpincl I miss the xmp PDF/UA metadata but hyperxmp copies the document title in the xmp data. Without hyperxmp it's just the opposite.

![minimal-pdftex](https://user-images.githubusercontent.com/11464744/74273551-f6950c00-4d10-11ea-8866-eb85258e46b7.png)
